config/drupal/phpcsfixer.rules.yml

```diff
@@ -24,28 +24,26 @@ parameters:
                 - try
                 - while
                 - yield
-        braces:
-            allow_single_line_closure: true
-            position_after_functions_and_oop_constructs: same
-            position_after_control_structures: same
-            position_after_anonymous_constructs: same
-        comment_to_phpdoc: false
+        curly_braces_position:
+            allow_single_line_anonymous_functions: true
+            allow_single_line_empty_anonymous_classes: true
+            classes_opening_brace: same_line
+            control_structures_opening_brace: same_line
+            functions_opening_brace: same_line
+        compact_nullable_type_declaration: true
+        # Keep compact_nullable_typehint until removed or change on
+        # https://github.com/drupol/phpcsfixer-configs-php
```

Comment:
It is removed in the dev version. Should we ask a new release instead?

Reply:
once a release of [drupol/phpcsfixer-configs-php](https://github.com/drupol/phpcsfixer-configs-php) we could remove, but let's keep it until a release will be made - not urgent.

Reply:
Ok, then I will ask for a new release because when testing I had tested on the dev version ;)

Reply:
okay, once done let's bump the verison of minimal required [/phpcsfixer-configs-php](https://github.com/drupol/phpcsfixer-configs-php) in this PR and remove my comment + line